DashAI/front/src/components/experiments/NewExperimentModal.jsx

```diff
@@ -40,11 +45,39 @@ const defaultNewExp = {
 export default function NewExperimentModal({ open, setOpen }) {
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.down("md"));
+  const { enqueueSnackbar } = useSnackbar();
 
   const [activeStep, setActiveStep] = useState(0);
   const [nextEnabled, setNextEnabled] = useState(false);
   const [newExp, setNewExp] = useState(defaultNewExp);
 
+  const uploadNewExperiment = async () => {
+    try {
+      await createExperimentRequest(
+        newExp.dataset.id,
+        newExp.task_name,
+        newExp.name,
+      );
+    } catch (error) {
+      enqueueSnackbar("Error while trying to create a new experiment", {
+        variant: "error",
+        anchorOrigin: {
+          vertical: "top",
+          horizontal: "right",
+        },
+      });
+
+      if (error.response) {
+        console.error("Response error:", error.message);
+      } else if (error.request) {
+        console.error("Request error", error.request);
+      } else {
+        console.error("Unkown Error", error.message);
+      }
+    } finally {
+      // setLoading(false);
```

Comment:
The [MUI button has a property to set it as loading](https://mui.com/material-ui/react-button/#loading-button). It is nice and keeps the user informed with respect to the request status.
I think that is great idea to replace the original button with the previous one.

pd: Sorry, i don't know where was the propper place to put this message than here ;)